Dockerfile

```diff
@@ -8,16 +8,14 @@ RUN  ls && npm run build
 
 FROM golang:1.17 as builder
 WORKDIR /app
-COPY ./ ./
```

Comment:
Moved this below so that code changes does not cause a redownload of all dependencies